homeassistant/components/ovo_energy/config_flow.py

```diff
@@ -53,7 +62,10 @@ async def async_step_user(self, user_input=None):
             step_id="user", data_schema=USER_SCHEMA, errors=errors
         )
 
-    async def async_step_reauth(self, user_input):
+    async def async_step_reauth(
+        self,
+        user_input: Mapping[str, Any],
+    ) -> FlowResult:
```

Comment:
```suggestion
    async def async_step_reauth(self, entry_data: Mapping[str, Any]) -> FlowResult:
        """Handle reauthorization request from Abode."""
        self.username = entry_data.get(CONF_USERNAME)

        return await self.async_step_reauth_confirm()

    async def async_step_reauth_confirm(
        self, user_input: dict[str, Any] | None = None
    ) -> FlowResult:
```

Reply:
I still don't understand why you are not applying this suggestion.
You should really split between reauth and reauth-confirm.